js/classes/entities/midground/weapon/slingshot.js

```diff
@@ -32,19 +40,34 @@ class Slingshot {
 
         // find the angle in radians from the x axis to the mouse
         const delta = Vector.worldToCanvasSpace(Vector.subtract(GAME.mousePos, this.pos));
-        // There is an undefined value here so I commented this out - CK
-        // console.log(GAME.mouse);
         let theta = Math.atan2(delta.y, delta.x);
         this.rotation = theta;
-        // There is an undefined value here so I commented this out - CK
 
         //TODO: swap animation frames based on rotation
 
-        console.log("rotation: " + " (" + this.rotation * 180 / Math.PI + " degrees)");
+        // console.log("rotation: " + " (" + this.rotation * 180 / Math.PI + " degrees)");
     }
 
-    fireSlingshot() {
-        ASSET_MGR.playAudio("./sfx/slingshot_launch.wav", 0.2);
+    fire() {
+        ASSET_MGR.stopAudio(SFX.SLINGSHOT_STRETCH.path);
+        
+        // choose from 4 different firing sounds
+        let rand = Math.floor(Math.random() * 4);
+        switch (rand) {
+            case 0:
+                ASSET_MGR.playAudio(SFX.SLINGSHOT_LAUNCH1.path, SFX.SLINGSHOT_LAUNCH1.volume);
+                break;
+            case 1:
+                ASSET_MGR.playAudio(SFX.SLINGSHOT_LAUNCH2.path, SFX.SLINGSHOT_LAUNCH2.volume);
+                break;
+            case 2:
+                ASSET_MGR.playAudio(SFX.SLINGSHOT_LAUNCH3.path, SFX.SLINGSHOT_LAUNCH3.volume);
+                break;
+            case 3:
+                ASSET_MGR.playAudio(SFX.SLINGSHOT_LAUNCH4.path, SFX.SLINGSHOT_LAUNCH4.volume);
+                break;
+        }
+
```

Comment:
I like that you play a random sound effect here! Also, notice that the only difference between each of the cases in your switch is the string used to index `SFX`. You could probably replace the entire switch with something like:
```
const sfx = SFX["SLINGSHOT_LAUNCH" + rand];
ASSET_MGR.playAudio(sfx.path, sfx.volume);
```